README.md

```diff
@@ -26,14 +26,16 @@ sequenceDiagram
 ### Usages
 
 The module is at the time of writing super opinionated in how the AWS IAM Open ID Connect Provider is created. The only
-thing the user needs to be concerned about is the `var.condition`. This needs to be set up the trust policy for
+thing the user needs to be concerned about is the `var.conditions`. This needs to be set up the trust policy for
 the `sub` field, which is explained
-more [here.](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#adding-the-identity-provider-to-aws)
+further [here.](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#adding-the-identity-provider-to-aws)
+
+Multiple fields apart from the required `sub` field are supported for more granular permissions, a reference to the contents of the OIDC token can be found [here.](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect#understanding-the-oidc-token)
 We currently don't intend supporting GitHub Environments. However, this might be changed in the future.
```

Comment:
If we switch to using a MAP instead of a string, we can narrow it down to specific environments. Therefore, we can remove the aforementioned statement 👍

Reply:
Done! 👍